website/site/content/docs/authentication-backends.md

```diff
@@ -41,8 +41,7 @@ To configure in Netlify CMS, use the same `backend` settings in your `config.yml
 
 ## GitHub Backend
 
-The GitHub backend allows CMS users to log in directly with their GitHub account. Note that the
-user's GitHub account must have push access to your content repository for this to work.
+For repositories stored on GitHub, the `github` backend allows CMS users to log in directly with their GitHub account. Note that the user's GitHub account must have push access to your content repository for this to work.
```

Comment:
Just reading this sounds like we repeat the word "GitHub" too much, but maybe it's necessary for disambiguation of what account the user is dealing with. 😄

Reply:
Heh, yeah. I think maybe we could remove the last "GitHub" and have it still be clear enough.